imod_coupler/drivers/dfm_metamod/dfm_metamod.py

```diff
@@ -141,8 +140,10 @@ def update(self) -> None:
         t_begin = self.get_current_time()
         self.mf6.prepare_time_step(0.0)
 
-        self.delt = self.mf6.get_time_step()
-        self.msw.prepare_time_step(self.delt)
+        self.dtgw = self.mf6.get_time_step()
+        self.dtsw = self.msw.get_sw_time_step()
+        self.number_substeps_per_modflowstep = int(self.dtgw / self.dtsw)
```

Comment:
it always need to be an integer, otherwise the metaswap kernel will crash (it expects an interger in self.msw.start_surface_water_time_step(idtsw)). i will raise an exception

Reply:
please raise the exception anyway if it's not an int. It improves clarity. when someone reads the code and doesn't know how metaswap works, this line will raise questions